app/controllers/sessions_controller.rb

```diff
@@ -27,12 +27,16 @@ def create
   def twitter
     # raise request.env["omniauth.auth"].to_yaml
     auth = request.env["omniauth.auth"]
+    if signed_in?
```

Comment:
So, by this logic, if the user is signed in, we should either find the user or create a new user? That doesn't work to well...